### PR TITLE
Login page merge ready 

### PR DESCRIPTION
**Pull request by b0llu**
image
image

### CODE REVIEW

**Review by gadgetfather — APPROVED** [review]
Good use of semantic HTML 👍

File: cart-page/cart.css [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+    display: flex;
+    justify-content: space-between;
+}
+
+.search-categories {
+    font-size: 1.3rem;
+    margin: 0 1rem;
+    padding: 1rem 0.8rem 1rem 1.4rem;
+    background: #fff;
+    border: 1px solid #ccc;
```
Comment (manojsarna) [inline cart-page/cart.css]: can use px here

File: cart-page/cart.css [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+.ammount-container {
+    align-self: flex-start;
+    margin: 0 2rem;
+}
+
+.ammount-container .card-container {
+    width: 50rem;
+    margin-top: 2rem;
+}
+
```
Comment (manojsarna) [inline cart-page/cart.css:50]: looks good!!

File: cart-page/cart.css [hunk 3]
```diff
@@ -0,0 +1,183 @@
...
+    .sidebar {
+        margin-left: 0;
+    }
+
+    .mobile-input {
+        display: block;
+        padding: 1rem;
+        border-radius: 0.5rem;
+        width: 100%;
+    }
```
Comment (manojsarna) [inline cart-page/cart.css]: can use px here

**Review by manojsarna — APPROVED** [review]
Looks Great , Clean Code

Can be merge :)

File: cart-page/cart.css (hunk 2 above)
Comment (b0llu) [reply cart-page/cart.css:50]: thanks

File: cart-page/cart.css (hunk 1 above)
Comment (b0llu) [reply cart-page/cart.css]: changed

File: cart-page/cart.css (hunk 3 above)
Comment (b0llu) [reply cart-page/cart.css]: changed

**Pull request merged**